src/Editor/EditorMain.vue

```diff
@@ -10,6 +10,9 @@
 
   <main>
     <div class="flex justify-center md:justify-end md:mr-[33px]">
+      <ButtonContainer>
+        <PenButton v-for="mode in getEnumValues(DrawModesEnum)" :key="mode" :pen-icon="mode" />
+      </ButtonContainer>
```

Comment:
Tą wyspe ja jeszcze wypozycjonuje to przy okazji dodam brakujacy przysk do zmiany algorytmow i tomenu rozwijane